tasks/main.yml

```diff
@@ -11,6 +11,11 @@
     enabled: yes
   when: systemd_networkd_network or systemd_networkd_link or systemd_networkd_netdev
 
+- name: Make sure systemd-resolved is installed
```

Comment:
Will need `become: true`.

Also, could you move it before `enable systemd-networkd`?

Reply:
done